src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/welcome-to-premium/page.tsx

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Link from "next/link";
+import buttonStyles from "../../../../../../../../components/server/button.module.scss";
+import styles from "./welcomeToPremium.module.scss";
+import { getL10n } from "../../../../../../../../functions/server/l10n";
+import { getLatestOnerepScanResults } from "../../../../../../../../../db/tables/onerep_scans";
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../../../../../../../api/utils/auth";
+import { getOnerepProfileId } from "../../../../../../../../../db/tables/subscribers";
+import { redirect } from "next/navigation";
+import { WelcomeToPremiumDonutChart } from "../../../../../../../../components/client/Chart";
+import { getSubscriberBreaches } from "../../../../../../../../functions/server/getUserBreaches";
+import { dashboardSummary } from "../../../../../../../../functions/server/dashboard";
+
+export default async function WelcomeToPremium() {
+  const l10n = getL10n();
+  const session = await getServerSession(authOptions);
+
+  if (!session?.user?.subscriber?.id) {
+    redirect("/redesign/user/dashboard/");
+  }
+
+  const result = await getOnerepProfileId(session.user.subscriber.id);
+  const profileId = result[0]["onerep_profile_id"] as number;
+  const scanResultItems =
+    (await getLatestOnerepScanResults(profileId))?.results ?? [];
+  const countOfDataBrokerProfiles = scanResultItems.length;
+  const subBreaches = await getSubscriberBreaches(session.user);
+  const summary = dashboardSummary(scanResultItems, subBreaches);
+  const totalExposures = summary.totalExposures;
+  const exposureReduction = Math.round(
+    (countOfDataBrokerProfiles / totalExposures) * 100
+  );
+
+  const chartData: Array<[string, number]> = [
+    ["reduction", exposureReduction],
+    ["Other", 100 - exposureReduction],
+  ];
+
+  return (
+    <>
+      <div>
+        <div className={styles.content}>
+          <h4>
+            {l10n.getString(
+              "welcome-to-premium-data-broker-profiles-title-part-one"
+            )}
+            <br />
+            {l10n.getString(
+              "welcome-to-premium-data-broker-profiles-title-part-two"
+            )}
+          </h4>
```

Comment:
Don't we have some Axe tests that check for violations and would catch invalid headings?

Reply:
Good question. We have the infra to easily run them in a unit test, but that test does need to get written for every state. But I also imagine that tests that verify heading levels are disabled there, since unit tests don't necessary test the full page.

Reply:
It might be something we can do at the Playwright level (if we think it's something we want vs just manual a11y testing).
https://playwright.dev/docs/accessibility-testing 
/cc @mozrokafor in case he has his $0.03.

Reply:
changed to h3